prow/prowjobs/GoogleCloudPlatform/testgrid/testgrid-images.yaml

```diff
@@ -13,3 +13,26 @@ presubmits:
         args:
         - test
         - //...
+
+postsubmits:
+  chases2/testgrid:
+  - name: push-testgrid-images
+    branches:
+    - ^master$
+    decorate: true
+    run_if_changed: '^cmd/.*$'
```

Comment:
This is not broad enough. If I change something in proto/ or metadata/ we'll want a new image with this change.

Reply:
It's not just the various protos or metadata; it's most of the repository.
I've changed this regex to reflect this.